tests/normal.rs

```diff
@@ -7,5 +7,5 @@
 )))]
 
 // Use the normal getrandom implementation on this architecture.
-use getrandom::getrandom as getrandom_impl;
+use getrandom::getrandom_uninit_slice as getrandom_impl;
```

Comment:
> Can we just keep this using the normal getrandom function?

I seem to remember needing to make this change because the common testing code needs to be able to call a public API or the internal `_impl` API with the same code. The `getrandom_uninit[_slice]` API's signature is nearly the same as the `_impl` functions.

That strategy of testing `getrandom` and assuming that it will cover `getrandom_uninit[_slice]` seems reasonable if/when `getrandom` is always implemented in terms of `getrandom_uninit[_slice]`, but that isn't guaranteed to always be the case. The optimal strategy for the `custom` feature is actually to have `getrandom_uninit[_slice]` implemented in terms of `getrandom` to avoid the duplicate zeroization I added in this PR. I am planning to do this removal of the duplicate zeroization in this way in the near future.

I think the right solution is to have tests that test both APIs.

Reply:
I think in the interest of keeping this PR as small as possible, we should keep the tests as-is (testing `getrandom_uninit` via `getrandom`) in this PR, and add additional testing in a followup PR.

Reply:
IIRC, the issue is with testing the rdrand backend. For now we could simply disable its testing and bring it back in a separate PR.

Reply:
Seems reasonable

Reply:
I just changed the tests back, and added a small shim in `rdrand.rs` to make everything build and pass.